config_defaults.yml

```diff
@@ -556,46 +556,43 @@ Rakefile:
 Gemfile:
   required:
     ':development':
-      # hardcode JSON version to what's shipped in pdk for now
-      - gem: json
-        version: '= 2.0.4'
-        condition: "Gem::Requirement.create('~> 2.4.2').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.1.0'
-        condition: "Gem::Requirement.create(['>= 2.5.0', '< 2.7.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.3.0'
-        condition: "Gem::Requirement.create(['>= 2.7.0', '< 2.8.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: 'puppet-module-posix-default-r#{minor_version}'
-        version: '~> 1.0'
+      - gem: 'json'
+      - gem: 'voxpupuli-puppet-lint-plugins'
+        version: '>= 3.0.0'
+      - gem: 'facterdb'
+        version: '< 2.0.0'
+      - gem: 'metadata-json-lint'
+        version: '< 4.0.0'
+      - gem: 'puppet-lint'
+        version: '< 3.0.0'
+      - gem: 'puppetlabs_spec_helper'
+        version: '< 4.0.0'
+      - gem: 'rspec-puppet'
+        version: '< 3.0.0'
+      - gem: 'rspec-puppet-facts'
+        version: '< 3.0.0'
+      - gem: 'codecov'
+      - gem: 'dependency_checker'
+      - gem: 'parallel_tests'
+      - gem: 'pry'
+      - gem: 'serverspec'
+      - gem: 'simplecov-console'
+      - gem: 'specinfra'
+      - gem: 'puppet-debugger'
+      - gem: 'ed25519'
         platforms: ruby
-      - gem: 'puppet-module-posix-dev-r#{minor_version}'
-        version: '~> 1.0'
+      - gem: 'bcrypt_pbkf'
         platforms: ruby
```

Comment:
`ed25519` and `bcrypt_pbkf` are needed by `net-ssh` but recent versions depend on them directly (see https://rubygems.org/gems/net-ssh). Really, they're only used for acceptance testing and can be dropped now.

Reply:
Yeh, have checked and they've both been added as dependencies for at least 5 years